decentraland/unity-explorer
# fix: World channel volume does not work in the build

## What does this PR change?
Fix #1395 

## How to test the changes?
1. Go to the top side of Genesis Plaza
2. Check that you hear the background music.
3. Go to the settings Menu -> audio section.
4. Modify the WORLD SFX slider to set it as 0.
5. Notice that the background music is being affected.
6. Do the the same with the sound of the doors when they open.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
